src/datetimepicker.test.jsx

```diff
@@ -202,4 +202,31 @@ describe('DateTimePicker', function() {
 		});
 
 	});
+
+	describe('onChangeCallback in props', () => {
+		let changeCallbackSpy;
+
+		beforeEach(() => {
+			changeCallbackSpy = spyOn(DateTimePicker.propTypes, 'onChangeCallback');
```

Comment:
this isn't spying on what you expect - the `propTypes` are functions that are called to validate props, not the actual values of the props.

You'll need to create a new spy, and I recommend using `jest.fn()`:

```jsx
const changeCallbackSpy = jest.fn();

<DateTimePicker name='start_time'
  value={dateStr}
  onChangeCallback={changeCallbackSpy}  // <-- supply the spy to the component
  forceCalendar
/>
```